autoload/lsp.vim

```diff
@@ -95,7 +95,7 @@ function! lsp#get_server_names() abort
 endfunction
 
 function! lsp#get_server_info(server_name) abort
-    return s:servers[a:server_name]['server_info']
+    return get(get(s:servers, a:server_name, {}), 'server_info', {})
```

Comment:
This is needed to pass test.

New implementation uses `get_server_info` in `lsp#omni#get_vim_completion_items` so I needed this...